Framework/Core/include/Framework/TableBuilder.h

```diff
@@ -664,8 +671,8 @@ class TableBuilder
 
   void validate() const;
 
-  template <typename... ARGS>
-  auto makeBuilders(std::array<char const*, sizeof...(ARGS)> const& columnNames, size_t nRows)
+  template <typename... ARGS, size_t I = sizeof...(ARGS)>
```

Comment:
Does this improve compilation time somehow? Is it not equivalent to `sizeof...` in the array declaration?

Reply:
What I see is that the sizeof...(ARGS) is left unexpanded (e.g. `sizeof...(int, float, ...)`), at least in some error messages, duplicating the already long list of template parameters. By having size_t I, you simply match on the size of the array, when possible, resulting in a much smaller template name.